Xamarin.Forms.Platform.iOS/CollectionView/ItemsViewController.cs

```diff
@@ -125,7 +119,24 @@ void CheckForEmptySource()
 		public override void ViewDidLoad()
 		{
 			base.ViewDidLoad();
-			AutomaticallyAdjustsScrollViewInsets = false;
+
+			ItemsSource = CreateItemsViewSource();
+			ItemsViewLayout.GetPrototype = GetPrototype;
+
+			Delegator = new UICollectionViewDelegator(ItemsViewLayout, this);
+			CollectionView.Delegate = Delegator;
+
+			if (!Forms.IsiOS11OrNewer)
+				AutomaticallyAdjustsScrollViewInsets = false;
+			else
+			{
+				// We set this property to keep iOS from trying to be helpful about insetting all the 
+				// CollectionView content when we're in landscape mode (to avoid the notch)
+				// The SetUseSafeArea Platform Specific is already taking care of this for us 
+				// That said, at some point it's possible folks will want a PS for controlling this behavior
+				CollectionView.ContentInsetAdjustmentBehavior = UIScrollViewContentInsetAdjustmentBehavior.Never;
```

Comment:
Moved this out of `ItemsViewRenderer` into here. `AutomaticallyAdjustsScrollViewInsets` was deprecated in iOS11. Now we use `ContentInsetAdjustmentBehavior`.